content/en/continuous_integration/setup_pipelines/gitlab.md

```diff
@@ -108,15 +108,15 @@ As an alternative to using the native Datadog integration, you can use [webhooks
 <div class="alert alert-info"><strong>Note</strong>: The native Datadog integration is the recommended approach and the option that is actively under development.</div>
 
 Go to **Settings > Webhooks** in your repository (or GitLab instance settings), and add a new webhook:
-* **URL**: `https://webhooks-http-intake.logs.datadoghq.com/v2/api/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][5].
+* **URL**: `https://webhooks-http-intake.logs.datadoghq.com/v2/api/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][4].
```

Comment:
This also needs a different domain for EU1 in the URL.

Reply:
Is it the same just replacing `datadoghq.com` with `datadoghq.eu`?

Reply:
yes